mempool/errors.go

```diff
@@ -59,18 +59,6 @@ func IsPreCheckError(err error) bool {
 	return errors.As(err, &ErrPreCheck{})
 }
 
-type ErrCheckTxAsync struct {
```

Comment:
Doesn't this make it Go API breaking? If so, I don't think we can merge this.

Reply:
This error file was merged into v0.38 two months ago. I'd say it's safe to remove it. #2277